safeeyes/safeeyes/config/lang/ru.json

```diff
@@ -0,0 +1,43 @@
+{
+    "meta_info": {
+        "language_name": "Русский",
+        "language_name_en": "Russian"
+    },
+    "exercises": {
+        "long_break_exercises": [
+            "Пройдитесь немного",
+            "Откиньтесь на спинку стула и расслабьтесь"
+        ],
+        "short_break_exercises": [
+            "Сильно зажмурьтесь",

```

Comment:
Its fine. I don't know even the abcd of Russian. I just wanna double check the possible human errors. Thank you for your response and contribution. I am merging it.